node/actors/executor/src/lib.rs

```diff
@@ -102,6 +98,21 @@ impl Executor {
         }
     }
 
+    /// Extracts a debug page config from http crate.
+    fn debug_page_config(&self) -> Option<http::DebugPageConfig> {
+        self.config
+            .debug_page_config
+            .as_ref()
+            .map(|debug_page_config| http::DebugPageConfig {
+                addr: debug_page_config.addr,
+                credentials: debug_page_config.credentials.clone(),
+                certs: http::load_certs(&debug_page_config.cert_path)
```

Comment:
that better, but please move loading and parsing even further away, so that executor doesn't need access to the file system